src/[...].test.js

```diff
@@ -41,12 +39,36 @@ describe('Testing create and delete', () => {
     const dom = new JSDOM('<ul class="list" id="list"></ul>');
     global.document = dom.window.document;
     create(tasks, addAllTasksToHTML, 'ok');
+    console.log(tasks);
```

Comment:
Kindly follow the [JS Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#javascript-in-the-browser) and do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.